### PR TITLE
updates for rails 5.2.2

### PR DESCRIPTION
**Pull request by ncliffor**

### CODE REVIEW

**Comment by ncliffor** [comment]
@ericsullivan do we want to remove support for older versions of Rails with Appraisals for Rails 5.2+?

**Comment by ncliffor** [comment]
feels like we could drop some of the gemfiles for older versions going forward

**Comment by ericsullivan** [comment]
Good point. We're likely safe dropping 3, 4.0, & 4.1

**Comment by ncliffor** [comment]
I figured if anyone needs support for those older versions they can just pin the gem anyway

File: dalliance.gemspec [hunk 1]
```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_dependency('rails', '>= 3.2', "< 5.2")
+  s.add_dependency('rails', '>= 3.2', "< 5.3")
```
Comment (ncliffor) [inline dalliance.gemspec]: s.add_dependency('rails', '>= 4.2', "< 5.3")

**Pull request merged**